client/influxdb.go

```diff
@@ -112,13 +99,21 @@ func (c *Client) Write(writes ...Write) (*Results, error) {
 	defer resp.Body.Close()
 
 	var results Results
-	dec := json.NewDecoder(resp.Body)
-	dec.UseNumber()
-	err = dec.Decode(&results)
+	body, _ := ioutil.ReadAll(resp.Body)
+	if len(body) > 0 {
```

Comment:
Is the response data always small? If there is any chance it could be large, don't use ReadAll, instead stream decode like was done before.